test/org/sagebionetworks/bridge/models/schedules/ScheduleTest.java

```diff
@@ -159,4 +159,30 @@ public void scheduleWithDelayNotPersistent() {
         schedule.setDelay((Period)null);
         assertTrue(schedule.getPersistent());
     }
+    
+    @Test
+    public void isOnceTaskWithoutTimes() {
+        Schedule schedule = new Schedule();
+        schedule.setScheduleType(ScheduleType.ONCE);
+        assertTrue(Schedule.isScheduleWithoutTimes(schedule));
+        
+        schedule.addTimes(LocalTime.parse("10:00"));
+        assertFalse(Schedule.isScheduleWithoutTimes(schedule));
+        
+        schedule.getTimes().clear();
+        schedule.setCronTrigger("some nonsense here");
+        assertFalse(Schedule.isScheduleWithoutTimes(schedule));
+        
+        schedule.setCronTrigger(null);
+        schedule.setScheduleType(ScheduleType.PERSISTENT);
+        assertTrue(Schedule.isScheduleWithoutTimes(schedule));
+    }
+
+    @Test
+    public void eventToPriorUTCMidnight() {
+        DateTime dateTime = DateTime.parse("2016-11-06T04:32.123-07:00");
+        DateTime midnight = Schedule.eventToMidnight(dateTime);
+        
+        assertEquals("2016-11-06T00:00:00.000Z", midnight.toString());
+    }
```

Comment:
Is it worth adding test cases that cross the date (late night with a negative offset that's tomorrow UTC, early morning with a positive offset that's yesterday UTC) to make sure we fully understand this behavior?